R/utils-ci.R

```diff
@@ -34,7 +34,7 @@ determine_license_env <- function(license) {
     cat_line("determine_license: looks like a license file")
     return(list(
       type = "file",
-      cmd_params = c("-v", paste0(license, ":/etc/rstudio-connect/license.lic")),
```

Comment:
This is the only change in this file, the rest is reformatting triggered on save.

Reply:
This is no longer true, I simplified a bit more when I switched from `docker-compose` to `docker`